packages/flutter/lib/src/widgets/widget_preview.dart

```diff
@@ -0,0 +1,75 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+/// Annotation used to mark functions that return widget previews.
+///
+/// {@tool snippet}
+///
+/// Functions annotated with `@Preview()` should return a `List<WidgetPreview>`
+/// and be public, top-level functions.
+///
+/// ```dart
+/// @Preview()
+/// List<WidgetPreview> myFirstPreview() {
```

Comment:
Instead of `myFirstPreview` should we standardize on a naming scheme for these functions and use it here? I imagine a lot of people who want a preview will just copy this block and then modify the body of the function.

Reply:
I don't know if we can standardize on a particular name since multiple previews can be defined in a library, but we can definitely make this snippet use a better name (`widgetPreview()` maybe?).